README.md

```diff
@@ -269,6 +269,16 @@ A `JsonApiClient::ResultSet` object should be paginatable with both `kaminari` a
 Person.where(name: 'Jeff').all
 ```
 
+## Headers
+
+You can specify the headers to send with all requests within a block with `with_headers`.
```

Comment:
This was already part of the `json_api_client` gem, it just wasn't documented. That said, I'll implement something similar for my `jsonapi-client` gem so I'm all ears if you have ideas on how to improve it.